introduction.md

```diff
@@ -1 +1,3 @@
-on with your favorite Code School path (Ruby, JavaScript, HTML & CSS, iOS or Electives)
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?